packages/object/src/index.ts

```diff
@@ -13,6 +13,7 @@ import { ObjectSet } from "./utils/objectSet.js";
 
 export * as ObjectPb from "./proto/drp/object/v1/object_pb.js";
 export * from "./hashgraph/index.js";
+import { clone, cloneDeep } from "es-toolkit";
```

Comment:
```suggestion
import { cloneDeep } from "es-toolkit";
```
remove unused import